.circleci/config.yml

```diff
@@ -87,6 +88,28 @@ jobs:
           command: |
             ~/.local/bin/tox -e codechecks
 
+  pyinstaller:
+    docker:
+      - image: "circleci/python:2"
```

Comment:
At first I thought I would comment here about using one of the pre-built Docker images to speed up the job.  But I see that the job only took 1m10s.  That's pretty good.  PyInstaller does pull in all of the Python dependencies too, though, right?  I wonder how it is so much faster than virtualenv/pip...

Reply:
Hmm, that _does_ seem rather fast. Could CircleCI have some sort of caching mechanism in place here? In any case, I've [added a `pip freeze` step](https://github.com/tahoe-lafs/tahoe-lafs/pull/661/commits/ee5a0d27bd58d16a7231b8f8c2e1e762b9d3668c) to the "pyinstaller" tox testenv and, as you can see in the ("Make PyInstaller executable") output [here](https://circleci.com/gh/tahoe-lafs/tahoe-lafs/13591) all of the expected dependencies are indeed in place.